src/quo2/components/navigation/page_nav.cljs

```diff
@@ -116,19 +129,74 @@
                     :margin-right (if push-to-the-left? 8 0))}
    [icons/icon icon (icon-props icon-color :big)]])
 
+(defn- render-right-section [right-section-buttons]
+  [rn/view {:style (assoc
+                    centrify-style
+                    :flex-direction :row
+                    :flex           1
+                    :justify-content :flex-end)}
+   (let [last-icon-index (- (count right-section-buttons) 1)]
+     (map-indexed (fn [index {:keys [icon background-color icon-color on-press]}]
+                    ^{:key index}
+                    [rn/touchable-opacity {:on-press on-press}
+                     [right-section-icon
+                      {:icon             icon
+                       :background-color background-color
+                       :icon-color       icon-color
+                       :push-to-the-left? (if (= index last-icon-index) false true)}]])
+                  right-section-buttons))])
+
 (defn page-nav
-  [{:keys [one-icon-align-left? horizontal-description? align-mid? page-nav-color page-nav-background-uri mid-section-type mid-section-icon mid-section-main-text mid-section-left-icon mid-section-right-icon mid-section-description mid-section-description-color mid-section-description-icon mid-section-description-user-icon mid-section-main-text-icon-color left-section-icon left-section-icon-color left-section-icon-background-color right-section-icons]}]
+  "[page-nav opts]
+   opts
+   { :one-icon-align-left? true/false
```

Comment:
[Code style] The docstring is not properly aligned and there are map keys with line breaks in the incorrect places. I'm mentioning this because automated formatting won't detect and fix this. I think the C-style braces is also too non-idiomatic in Clojure.

[Random comment]
To be honest, I know the codebase is adding such docstrings in some components, but I'm not a fan at all.

1. They easily drift from the source code. The one in this PR is already outdated/incorrect, e.g. the `:icon-background-color` key doesn't have a a value).
2. They can't be automatically verified.
3. The types can't be reinforced.

What I think we should do is spec (with clojure.spec, Malli or another tool) and enable instrumentation during development for highly reusable components, such as the ones in quo2. Of course, this is a much bigger discussion, not in this PR's scope.

Reply:
Completely agree on all points here. We should bring this discussion to the team and for now I will adjust this docstring and fix the formatting on it etc.

Reply:
Thanks @J-Son89 :)